### ISSUE TITLE
XLWorkbook SaveAs(memoryStream) Error : The given key '80' was not present in the dictionary.

### ISSUE THREAD

**Issue by Vampire-V** [ClosedXML/ClosedXML#1995]
## Read and complete the full issue template

**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature
- [ ] Question

**Did you test against the latest CI build?**
- [ ] Yes
- [x] No

**Code to reproduce problem:**
```c#
public void Main()
{
    using (var ms = new MemoryStream())
            {
               // file = "IFormFile" user import
                await file.CopyToAsync(ms);
                var wb = new XLWorkbook(ms);
                wb.SaveAs(ms);
            }
}
```

- [x] I attached a sample spreadsheet.  (You can drag files on to this issue)
![image](https://user-images.githubusercontent.com/25393048/215406708-fe961cd6-085f-4fd9-843d-4a56c4e27468.png)
[PRO_VL_JAN23_Big Bear (New File).xlsx](https://github.com/ClosedXML/ClosedXML/files/10533193/PRO_VL_JAN23_Big.Bear.New.File.xlsx)

Error : 
![image](https://user-images.githubusercontent.com/25393048/215407439-430294b7-c16c-473d-8d49-289043c4283a.png)

**Comment by jahav** [comment]
This has been fixed in 0.100, I can load and save the provided file with 0.100.0, but not with 0.97.

**Comment by jahav** [comment]
Duplicate of #1884
